icons/tauri/tauri-plain.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" fill="rgb(5.882353%,5.882353%,5.882353%)"><path d="M86.148 46.84c0 6.703-5.434 12.141-12.137 12.141s-12.14-5.438-12.14-12.141a12.14 12.14 0 0 1 12.141-12.137A12.14 12.14 0 0 1 86.148 46.84zM41.457 81.598a12.14 12.14 0 0 1 12.141-12.137 12.14 12.14 0 0 1 12.137 12.137 12.14 12.14 0 0 1-12.137 12.141 12.14 12.14 0 0 1-12.141-12.141zm0 0"/><g fill-rule="evenodd"><path d="M99.164 85.766a46.29 46.29 0 0 1-15.988 6.504c1.16-3.344 1.789-6.934 1.789-10.672 0-1.355-.082-2.691-.242-4.004a32.58 32.58 0 0 0 6.965-3.402c5.266-3.402 9.438-8.258 12.016-13.973a32.58 32.58 0 0 0 2.504-18.262c-.937-6.199-3.648-11.996-7.805-16.695a32.61 32.61 0 0 0-15.621-9.785 32.59 32.59 0 0 0-18.43.262c-5.988 1.859-11.312 5.41-15.336 10.223-.512.617-1.004 1.25-1.469 1.898a54.16 54.16 0 0 0-17.773 5.164A46.28 46.28 0 0 1 60.27 2.582a46.3 46.3 0 0 1 26.223-.371c8.598 2.402 16.313 7.238 22.227 13.922s9.773 14.934 11.113 23.758a46.38 46.38 0 0 1-3.566 25.984 46.37 46.37 0 0 1-17.102 19.891zm-68.84-44.289l11.363 1.395c.27-2.203.762-4.359 1.465-6.441a46.17 46.17 0 0 0-12.828 5.047zm0 0"/><path d="M28.359 42.676a46.45 46.45 0 0 1 16.117-6.531c-1.289 3.504-1.992 7.293-1.992 11.25 0 1.18.063 2.348.184 3.5a32.49 32.49 0 0 0-6.828 3.355c-5.266 3.402-9.441 8.254-12.016 13.973a32.58 32.58 0 0 0-2.504 18.262c.938 6.199 3.648 11.996 7.805 16.695s9.582 8.094 15.621 9.781a32.54 32.54 0 0 0 18.426-.258 32.58 32.58 0 0 0 15.34-10.223c.516-.621 1.008-1.258 1.477-1.91a54.04 54.04 0 0 0 17.781-5.199 46.33 46.33 0 0 1-8.687 15.941 46.32 46.32 0 0 1-21.824 14.547 46.3 46.3 0 0 1-26.223.371c-8.598-2.402-16.316-7.238-22.23-13.922A46.38 46.38 0 0 1 7.695 88.551c-1.336-8.824-.098-17.848 3.566-25.988s9.605-15.043 17.098-19.887zm68.777 44.281a2.67 2.67 0 0 1-.18.102 2.67 2.67 0 0 0 .18-.102zm0 0"/></g></svg>
```

Comment:
For plain and line icons, please merge the paths into one path if possible

ref: https://github.com/devicons/devicon/wiki/SVG-Standards

Reply:
done with https://github.com/devicons/devicon/pull/1457/commits/7b0f4c6cee29cdde9dfbee6c9f69c4e409405125